projects/js-packages/publicize-components/src/components/media-picker/index.js

```diff
@@ -75,15 +80,30 @@ export default function MediaPicker( {
 								duration={ length }
 							></VideoPreview>
 						) : (
-							<ResponsiveWrapper naturalWidth={ width } naturalHeight={ height } isInline>
-								<img src={ sourceUrl } alt="" className={ styles[ 'preview-image' ] } />
+							<ResponsiveWrapper
+								naturalWidth={ previewWidth || width }
+								naturalHeight={ previewHeight || height }
+								isInline
+							>
+								<img src={ previewUrl } alt="" className={ styles[ 'preview-image' ] } />
```

Comment:
Should we also use `sourceUrl` here if we're also falling back to `width` and `height`?

```suggestion
								<img src={ previewUrl || sourceUrl } alt="" className={ styles[ 'preview-image' ] } />
```